arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -625,7 +625,7 @@ public void traceContextEnter(MessageFrame frame) {
           frameType,
           newChildContextNumber(),
           this.deploymentStatusOf(frame.getContractAddress()),
-          frame.getValue(),
+          frame.getApparentValue(),
```

Comment:
Important bug fix, required e.g. for DELEGATECALL to provide the CallFrame with the correct `value`.